mods/ts/rules/defaults.yaml

```diff
@@ -1084,6 +1084,23 @@
 	EditorTilesetFilter:
 		Categories: Tunnel
 
+^Waterfall:
+	EditorOnlyTooltip:
+		Name: Waterfall Animation
+	RenderSprites:
+		Palette: terrain
+	WithSpriteBody:
+	BodyOrientation:
+		UseClassicPerspectiveFudge: False
+		QuantizedFacings: 1
+	Building:
+		Dimensions: 1, 1
```

Comment:
IMO this should be set to match the actual size of the waterfall - e.g. 3x1.

Reply:
That will make the correct placement impossible. Similar to the mess we currently have with tunnel tops in the map editor.

Reply:
I ended up having to redo the tunnel top definitions to give them a proper footprint, like this, to solve that mess.

Reply:
Actually the reference point is a 1x1 cell here. Compare the first frame of the waterfall, which is a small square from neighboring cliffs etc.

Reply:
The animations should freeze when under the fog, and `FrozenUnderFog` requires a proper footprint to be set up.

Reply:
Actually `FrozenUnderFog` is overkill - `HiddenUnderFog` would have less performance overhead and do the job just as well.  In any case, that also requires a proper footprint.

Reply:
But how do you solve the map editor placement. It will always glitch due to heights and I assume the footprint which is too large. Try importing "They All Float" and place the tunnel top definitions. Some, actually most, will be impossible to get right as you can't position them at the right cell.

Reply:
By fixing the actor placement tool in the editor to handle the edge case where the TopLeft cell is at a different height to the cell underneath the cursor.